aimmo-game/simulation/maps/cell.py

```diff
@@ -0,0 +1,58 @@
+from simulation.action import MoveAction
+
+
+class Cell(object):
+    """
+    Any position on the world grid.
+    """
+
+    def __init__(self, location, habitable=True, generates_score=False, partially_fogged=False):
+        self.location = location
+        self.habitable = habitable
+        self.generates_score = generates_score
+        self.avatar = None
+        self.pickup = None
+        self.partially_fogged = partially_fogged
```

Comment:
so are we keeping this for the moment?

Reply:
If we're on a removing code mission for the minimal lovable product, I think it could go away as well :)

Reply:
Yup. Will need to do it in another PR though because these instance variables are referenced everywhere.

Reply:
#248